build.rs

```diff
@@ -0,0 +1,14 @@
+use anyhow::{Ok, Result};
+use substreams_ethereum::Abigen;
+
+fn main() -> Result<(), anyhow::Error> {
+    // Abigen::new("CollectionV2", "abi/collectionV2.json")?
+    //     .generate()?
+    //     .write_to_file("src/abi/collectionV2.rs")?;
+
+    Abigen::new("CollectionFactoryV3", "abi/CollectionFactoryV3.json")?
+        .generate()?
+        .write_to_file("src/abi/collectionFactoryv3.rs")?;
+
```

Comment:
You can add extra steps here to generate the `modified` version of `collectionV2`